Auth0Tests/CredentialsManagerErrorSpec.swift

```diff
@@ -80,45 +80,48 @@ class CredentialsManagerErrorSpec: QuickSpec {
-                let message = "No valid credentials found."
+                let message = "No credentials were found in the store."
                 let error = CredentialsManagerError(code: .noCredentials)
                 expect(error.localizedDescription) == message
             }
 
             it("should return message for no refresh token") {
-                let message = "No Refresh Token in the credentials."
+                let message = "The stored credentials instance does not contain a Refresh Token."
                 let error = CredentialsManagerError(code: .noRefreshToken)
                 expect(error.localizedDescription) == message
             }
 
-            it("should return message for PKCE not allowed") {
-                let minTTL = 7200
-                let lifetime = 3600
-                let message = "The minTTL requested (\(minTTL)s) is greater than the "
-                + "lifetime of the renewed Access Token (\(lifetime)s). Request a lower minTTL or increase the "
-                + "'Token Expiration' setting of your Auth0 API in the dashboard."
-                let error = CredentialsManagerError(code: .largeMinTTL(minTTL: minTTL, lifetime: lifetime))
-                expect(error.localizedDescription) == message
-            }
-
-            it("should return the default message for refresh failed") {
-                let message = "Failed to perform Credentials Manager operation."
+            it("should return message for renew failed") {
+                let message = "The credentials renewal failed. The underlying 'AuthenticationError' can be"
+                + " accessed via the 'cause' property."
                 let error = CredentialsManagerError(code: .renewFailed)
                 expect(error.localizedDescription) == message
             }
 
-            it("should return the default message for biometrics failed") {
-                let message = "Failed to perform Credentials Manager operation."
+            it("should return message for biometrics failed") {
+                let message = "The Biometric authentication failed. The underlying 'LAError' can be accessed"
+                + " via the 'cause' property."
                 let error = CredentialsManagerError(code: .biometricsFailed)
                 expect(error.localizedDescription) == message
             }
 
-            it("should return the default message for revoke failed") {
-                let message = "Failed to perform Credentials Manager operation."
+            it("should return message for revoke failed") {
+                let message = "The revocation of the Refresh Token failed. The underlying 'AuthenticationError'"
+                + " can be accessed via the 'cause' property."
                 let error = CredentialsManagerError(code: .revokeFailed)
                 expect(error.localizedDescription) == message
             }
 
+            it("should return message for PKCE not allowed") {
```

Comment:
`should return message when minTTL is too big`?

Reply:
Good catch! Fixed in https://github.com/auth0/Auth0.swift/pull/603/commits/7d3f6435b95f29c42c8d9c680847f23a6b3df17f. Thanks